src/pages/dataSetsWip/Edit.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react'
+import { useQuery } from 'react-query'
+import { useParams } from 'react-router-dom'
+import {
+    FormBase,
+    SectionedFormLayout,
+    DefaultSectionedFormFooter,
+    DefaultSectionedFormSidebar,
+    SectionedFormErrorNotice,
+} from '../../components'
+import { SectionedFormProvider, SECTIONS_MAP, useOnSubmitEdit } from '../../lib'
+import { useBoundResourceQueryFn } from '../../lib/query/useBoundQueryFn'
+import { PickWithFieldFilters, DataSet } from '../../types/generated'
+import { DataSetFormContents } from './form/DataSetFormContents'
+import { validate } from './form/dataSetFormSchema'
+import { DataSetFormDescriptor } from './form/formDescriptor'
+
+const section = SECTIONS_MAP.dataSet
+
+const fieldFilters = [
+    'id,displayName,dataSetElements[dataElement[id,displayName],categoryCombo[id]],style[color,icon],indicators[id,displayName]',
+] as const
+type DataSetValues = PickWithFieldFilters<DataSet, typeof fieldFilters>
+
+export const Component = () => {
```

Comment:
@Birkbjo added this so that we could check within the dataSetsWIP in the new app that the changes were being saved 🙏 (noting because it's not required for the ticket requirements)